api/src/opentrons/protocol_engine/commands/load_pipette.py

```diff
@@ -46,22 +57,32 @@ class LoadPipetteResult(BaseModel):
 
 
 class LoadPipetteImplementation(
-    AbstractCommandImpl[LoadPipetteParams, LoadPipetteResult]
+    AbstractCommandWithPrivateResultImpl[
+        LoadPipetteParams, LoadPipetteResult, LoadPipettePrivateResult
+    ]
 ):
     """Load pipette command implementation."""
 
     def __init__(self, equipment: EquipmentHandler, **kwargs: object) -> None:
         self._equipment = equipment
 
-    async def execute(self, params: LoadPipetteParams) -> LoadPipetteResult:
+    async def execute(
+        self, params: LoadPipetteParams
+    ) -> Tuple[LoadPipetteResult, LoadPipettePrivateResult]:
         """Check that requested pipette is attached and assign its identifier."""
         loaded_pipette = await self._equipment.load_pipette(
             pipette_name=params.pipetteName,
             mount=params.mount,
             pipette_id=params.pipetteId,
         )
 
-        return LoadPipetteResult(pipetteId=loaded_pipette.pipette_id)
+        return LoadPipetteResult(
+            pipetteId=loaded_pipette.pipette_id
+        ), LoadPipettePrivateResult(
+            pipette_id=loaded_pipette.pipette_id,
+            serial_number=loaded_pipette.serial_number,
+            config=loaded_pipette.static_config,
+        )
```

Comment:
Nitpick for slightly faster analysis:
 
```suggestion
        return LoadPipetteResult.construct(
            pipetteId=loaded_pipette.pipette_id
        ), LoadPipettePrivateResult.construct(
            pipette_id=loaded_pipette.pipette_id,
            serial_number=loaded_pipette.serial_number,
            config=loaded_pipette.static_config,
        )
```